src/Benchmarks.ServerJob/WebHost.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Benchmarks.ServerJob
+{
+    public enum WebHost
+    {
+        Kestrel,
+        WebListener,
```

Comment:
Since `Microsoft.AspNetCore.Server.WebListener` has been renamed to `Mcrosoft.AspNetCore.Server.HttpSys`, I think we should replace `WebListener` with `HttpSys` in your new code and also Benchmarks itself.

Rename announcement: https://github.com/aspnet/Announcements/issues/211